components/webac/src/main/java/org/trellisldp/webac/Authorization.java

```diff
@@ -142,4 +143,13 @@ public BlankNodeOrIRI getIdentifier() {
     public Set<IRI> getDefault() {
         return unmodifiableSet(dataMap.get(ACL.default_));
     }
+
+    /**
+     * Retrieve the acceptable origins of the ACL document.
+     *
+     * @return the origin IRIs
+     */
+    public Set<IRI> getOrigin() {
```

Comment:
If this gets called more than once and since `dataMap` doesn't seem to change it might be worth building this `Set<IRI>` and keeping it in a field to avoid building short-lived objects.

Reply:
That's an excellent point. I reworked the constructor to `.collect` the values directly into immutable `Set<IRI>` objects so that there is no need to create new objects.